sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/aio/_transport_async.py

```diff
@@ -427,35 +439,81 @@ def __init__(self, host, port=WEBSOCKET_PORT, connect_timeout=None, ssl=None, **
         self._custom_endpoint = kwargs.get("custom_endpoint")
         self.host = host
         self.ws = None
-        self._http_proxy = kwargs.get('http_proxy', None)
+        self.session = None
+        self._http_proxy = kwargs.get("http_proxy", None)
 
     async def connect(self):
         http_proxy_host, http_proxy_port, http_proxy_auth = None, None, None
+        ssl_opts: Optional[SSLContext] = self._build_ssl_opts(self.sslopts) if len(self.ssl_opts) > 0 else None
```

Comment:
```suggestion
        ssl_opts: Optional[SSLContext] = self.sslopts if len(self.sslopts) > 0 else None
```

Reply:
I think self.sslopts is built from _build_ssl_opts on line 183

Reply:
good catch, it is